README.md

```diff
@@ -397,9 +401,9 @@ After updating the stack, use `Get-AzSubscriptionDeploymentStack` to list the re
 
 ## Use snapshots
 
-Snapshots provide a way to view the history of updates to the stack. The resource type is read-only. New snapshots are created by virtue of updating the parent stack. Snapshots are primarily used for viewing history for diagnostics or troubleshooting. In the future, you can roll back a stack to a previous snapshot if there is an error.
+Snapshots provide a way to view the history of updates to the stack. A snapshot is read-only and is created whenever any template resource is successfully deployed.  Note that not all resources need to be successfully deployed for a snapshot to be created. Snapshots are primarily used for viewing history for diagnostics or troubleshooting. In the future, a snapshot can be applied to roll back a deploymentStack to a previous state if there is an error.
```

Comment:
I made an edit - can you confirm this is accurate:

```A snapshot is read-only and is created whenever any template resource is successfully deployed. Note that not all resources need to be successfully deployed for a snapshot to be created.```

Reply:
Noteworthy distinction, good add